__init__.py

```diff
@@ -310,12 +303,29 @@ def execute(self, _context):
         print("Starting Import")
         import webbrowser
         print("Starting Login")
-        if thangs_api.bearer == "":
+        
+        bearer_DIR = os.path.join(os.path.dirname(__file__), 'bearer.json')
+        if not os.path.exists(bearer_DIR):
+            print("Creating Bearer.json")
+            f = open(bearer_DIR, "x")
+        
+        # check if size of file is 0
+        if os.stat(bearer_DIR).st_size == 0:
+            print("Json was empty")
             thangs_login.startLoginFromBrowser()
             print("Waiting on Login")
             thangs_login.token_available.wait()
-            fetcher.bearer = str(thangs_login.token["TOKEN"])
-            thangs_api.bearer = str(thangs_login.token["TOKEN"])
+            bearer = {
+                'Bearer': str(thangs_login.token["TOKEN"]),

```

Comment:
json conventions are generally that keys are lower case.  So, let's make this `bearer` instead of `Bearer`